src/components/search_bar/search_filters.tsx

```diff
@@ -30,7 +30,7 @@
 
 import React, { Component, Fragment, ReactElement } from 'react';
 import { createFilter, SearchFilterConfig } from './filters';
-import { Query } from './query';
+import { Query } from './query/query';
```

Comment:
`Typescript` doesn't want to export the `Query` class from the `index.ts` for reasons I don't understand, so I had to adjust the imports